pkg/apis/harvesterhci.io/v1beta1/image.go

```diff
@@ -57,6 +58,12 @@ type VirtualMachineImageSpec struct {
 
 	// +optional
 	StorageClassParameters map[string]string `json:"storageClassParameters"`
+
+	// +optinoal
+	// +kubebuilder:default:=3
+	// +kubebuilder:validation:Minimum:=0
+	// +kubebuilder:validation:Optional
+	Retry int `json:"retry" default:"3"`
```

Comment:
Question: should this retry have a maximum limit ? to avoid potential endless retry

Reply:
Do you mean add something like `+kubebuilder:validation:Maximum:=10` on it?

Reply:
yes, with an proper up limit

Reply:
Updated it. Thanks.